xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/resources/document-formats.js

```diff
@@ -19,196 +19,666 @@
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Windows Bitmap",
+    "extensions": [
+      "bmp"
+    ],
+    "mediaType": "image/bmp",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_bmp_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_bmp_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Graphics Interchange Format",
+    "extensions": [
+      "gif"
+    ],
+    "mediaType": "image/gif",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_gif_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_gif_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Joint Photographic Experts Group",
+    "extensions": [
+      "jpg",
+      "jpeg"
+    ],
+    "mediaType": "image/jpeg",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_jpg_Export"
+      },
+      "SPREADSHEET": {
+        "FilterName": "calc_jpg_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_jpg_Export"
+      },
+      "TEXT": {
+        "FilterName": "writer_jpg_Export"
+      },
+      "WEB": {
+        "FilterName": "writer_web_jpg_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Portable Network Graphics",
+    "extensions": [
+      "png"
+    ],
+    "mediaType": "image/png",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_png_Export"
+      },
+      "SPREADSHEET": {
+        "FilterName": "calc_png_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_png_Export"
+      },
+      "TEXT": {
+        "FilterName": "writer_png_Export"
+      },
+      "WEB": {
+        "FilterName": "writer_web_png_Export"
+      }
+    }
   },
   {
     "name": "Scalable Vector Graphics",
-    "extensions": ["svg"],
+    "extensions": [
+      "svg"
+    ],
     "mediaType": "image/svg+xml",
     "inputFamily": "DRAWING",
-    "storeProperties": {"DRAWING": {"FilterName": "draw_svg_Export"}}
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_svg_Export"
+      },
+      // New properties
+      "SPREADSHEET": {
+        "FilterName": "calc_svg_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_svg_Export"
+      },
+      "TEXT": {
+        "FilterName": "writer_svg_Export"
+      },
+      "WEB": {
+        "FilterName": "writer_svg_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Tagged Image File Format",
+    "extensions": [
+      "tif",
+      "tiff"
+    ],
+    "mediaType": "image/tiff",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_tif_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_tif_Export"
+      }
+    }
+  },
+  // Missing from the default file
```

Comment:
The property below is not in the default jodconverter file I assume it's some custom property we provided. Sounds legit to keep it.

Reply:
I don't recall adding this either. It was probably dropped from the standard document formats file.

Reply:
This format was part of the library, but was removed in https://github.com/jodconverter/jodconverter/commit/fc6f56dff377b8da582023c224f142868ba0eccc#diff-43716e794ac44a816b4c7145455f9e97b0a82b63cb20d421684fa3297a10d8f3.